From 80010acd052ca7fe544740144756cf9fc2fad629 Mon Sep 17 00:00:00 2001
From: Jakub Jelinek <jakub@redhat.com>
Date: Thu, 24 Nov 2022 10:33:00 +0100
Subject: [PATCH 423/547] testsuite: Fix up broken testcase [PR107127]

I've added { dg-options "" } line manually in the patch but
forgot to adjust the number of added lines.

2022-11-24  Jakub Jelinek  <jakub@redhat.com>

	PR c/107127
	* gcc.dg/pr107127.c (foo): Add missing closing }.

(cherry picked from commit add0f941be18cdf962a0f300019acacbf2325d41)
---
 gcc/testsuite/gcc.dg/pr107127.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/testsuite/gcc.dg/pr107127.c b/gcc/testsuite/gcc.dg/pr107127.c
index f7ae87db70e..d25cced6b33 100644
--- a/gcc/testsuite/gcc.dg/pr107127.c
+++ b/gcc/testsuite/gcc.dg/pr107127.c
@@ -10,3 +10,4 @@ foo (_Complex double a, double b, double c)
   return v[0] / ((((c * (0 - 0 / a + 699.0 + 7.05 - 286.0 - +-4.65 + 1.57 + 0) * 0.1 - 3.28 + 4.22 + 0.1)) * b + 5.06)
 		 * 1.23 * 8.0 * 12.0 * 16.0 * 2.0 * 2.0 * 0.25 * 0.125 * 18.2 * -15.25 * 0.0001
 		 * 42.0 * 0.012 - 8.45 + 0 + 88.0 + 6.96 + 867.0 + 9.10 - 7.04 * -1.0);
+}
-- 
2.25.1

